src/main/scala/jigg/util/JSONUtil.scala

```diff
@@ -35,27 +42,24 @@ object JSONUtil {
     def hasAttribution(x: Node): Boolean = !x.attributes.isEmpty
   }
 
-  def toJSON(x: Any): StringBuilder = x match {
-    case x: String if x.endsWith(".xml")    => toJSONFromNode(XML.loadFile(x))
-    case x: Elem      => toJSONFromNode(x)
-    case _            => new StringBuilder("")
-  }
+  def readJSON(file: String): String = toJSONFromNode(XML.loadFile(file))
```

Comment:
The `readJSON` method is not needed in this situation.
I'll remove it from `JSONUtil`